web/src/main/java/com/futurewei/alcor/web/entity/gateway/GatewayEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.futurewei.alcor.web.entity.gateway;
+
+
+import com.futurewei.alcor.common.entity.CustomerResource;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class GatewayEntity extends CustomerResource {
+    private GatewayType type;
+    private String status; // (available)
+    private List<GatewayIp> ips;
+    private List<String> attachments;
+    private List<String> routetables;
+    private List<String> tags;
+    private String owner;
+    private Map<String, String> options;
+}
```

Comment:
@songxiaoyan Do we need constructor here?

Reply:
you mean it should be constructor in the GM entity package?

Reply:
I mean such the code like this:
```
public GatewayEntity() {
    }

    public GatewayEntity(GatewayType type, String status, List<GatewayIp> ips, private List<String> attachments, List<String> routetables, ...) {
... 
}
```

Reply:
oh,I get your mean,I use the set method to assign values, if necessary, I can add the construction method